### PR TITLE
Feature/frontend/advanced settings

### PR DESCRIPTION
**Pull request by nakajima-john-shotaro**
## Pull Request Type

- [ ] :bug: `fix` : (bug fix for the user, not a fix to a build script)
- [ ] :adhesive_bandage: `change` : (simple fix for a non-critical issue)
- [x] :sparkles: `feat` : (new feature for the user, not a new feature for build script)
- [ ] :memo: `docs` : (changes to the documentation)
- [ ] :recycle: `refactor` : (refactoring production code, eg. renaming a variable)
- [ ] :pencil2: `style` : (formatting, missing semi colons, etc; no production code change)
- [ ] :rotating_light: `test` : (adding missing tests, refactoring tests; no production code change)
- [ ] :package: `chore` : (updating grunt tasks etc; no production code change)
- [ ] :rewind: `reverts` : (canceling commits or pull requests)

## Description

Add some functions.

## Pre-Review Checklist for the Pull Request Author

- [x] Code follows coding-guidelines
- [x] Assign Pull Request to reviewer

## Checklist for the Pull Request Reviewer

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows coding-guidelines
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] Pull Request title describes the changes

## Post-Review Checklist for the Pull Request Author

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write release notes

### CODE REVIEW

File: aicon/frontend/static/css/aicon.css [hunk 1]
```diff
@@ -1,3 +1,12 @@
+/* フォントの設定 */
+@font-face {
+	font-family: 'MyFont';
```
Comment (urasakikeisuke) [inline aicon/frontend/static/css/aicon.css:3]: もう少しユニークな名前はどうですか？
```suggestion
	font-family: 'NicoKaku';

```

File: aicon/frontend/static/css/aicon.css [hunk 2]
```diff
@@ -1,3 +1,12 @@
+/* フォントの設定 */
+@font-face {
+	font-family: 'MyFont';
+	src: url(../font/nicokaku_v1.ttf);
+}
+
+#nico_font {
+	font-family: MyFont;
```
Comment (urasakikeisuke) [inline aicon/frontend/static/css/aicon.css:8]:
```suggestion
	font-family: NicoKaku;
```

**Review by urasakikeisuke — CHANGES_REQUESTED** [review]

File: aicon/frontend/static/css/aicon.css (hunk 1 above)
Comment (nakajima-john-shotaro) [reply aicon/frontend/static/css/aicon.css:3]: はい、そうさせていただきます。🐬

**Review by urasakikeisuke — APPROVED** [review]
ThankYouJoeBidenGIF

**Pull request merged**